tests/TestReaderJSON.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+
+from Commons.ReaderJSON import ReaderJSON
+
+
+class TestReaderJSON(unittest.TestCase):
+    path: list
+    json: dict
+
+    reader_channels: ReaderJSON
+    reader_video: ReaderJSON
+
+    open_channels: str
+    open_video: str
+
+    json_channels: dict
+    json_video: dict
+
+    def setUp(self) -> None:
+        self.reader_channels = ReaderJSON('YouTube/Lake/jsonTypesFile/YouTube/dataVideos.json')
```

Comment:
It would be better to use raw colection or static data here without your code parts